internal/database/metadata/informer/feature.go

```diff
@@ -34,9 +34,9 @@ func (c *FeatureCache) List(opt metadata.ListFeatureOpt) types.FeatureList {
 	}
 
 	// filter names
-	if opt.FeatureFullNames != nil {
+	if opt.FeatureNames != nil {
 		var tmp types.FeatureList
-		for _, fullName := range *opt.FeatureFullNames {
+		for _, fullName := range *opt.FeatureNames {
```

Comment:
Does it seem more consistent to use the name here? When for-loop shortNames, use shortName

Reply:
@lianxmfor make sense